### ISSUE TITLE
Missing exported method due to bundle size increase

### ISSUE THREAD

**Issue by kochis** [maplibre/maplibre-style-spec#56]
Is there a reason functions like `format` and `validate` aren't included in the default export? Seems to be how it's done for `mapbox-gl-style-spec` so curious if that's intended or not.

```js
import { format, validate } from '@maplibre/maplibre-gl-style-spec';
```
<img width="1544" alt="image" src="https://user-images.githubusercontent.com/814934/224868039-86a35242-8b1b-4875-babd-01bda27fd4e6.png">

I had to end up doing the following to get validation to work (this is client-side code)
```js
import * as StyleSpec from '@maplibre/maplibre-gl-style-spec';
...
const errors = StyleSpec.validateStyleMin(styleObj);
```
Also doesn't seem like the `format` function is being exported either.

**Comment by HarelM** [comment]
When we exported them here it causes the maplibre-gl-js package size to increase, so as a temporary solution, we removed them from the export.

**Comment by kochis** [comment]
>  so as a temporary solution, we removed them from the export.

Got it. Is it recommended to stick with the mapbox version for the time being if these are needed?

**Comment by HarelM** [comment]
You can use an earlier version of maplibre spec I believe. You call.

**Comment by HarelM** [comment]
Duplicate of #24.
Resolved by #464.
